src/tr/expand.rs

```diff
@@ -88,14 +88,19 @@ impl<'a> Iterator for ExpandSet<'a> {
 
         if let Some(first) = self.unesc.next() {
             // peek ahead
-            if self.unesc.peek() == Some(&'-') && match self.unesc.size_hint() {
-                (x, _) if x > 1 => true, // there's a range here; record it in our internal Range struct
-                _ => false,
-            } {
+            if self.unesc.peek() == Some(&'-')
+                && match self.unesc.size_hint() {
+                    (x, _) if x > 1 => true, // there's a range here; record it in our internal Range struct
+                    _ => false,
+                }
+            {
                 self.unesc.next(); // this is the '-'
                 let last = self.unesc.next().unwrap(); // this is the end of the range
 
-                self.range = first as u32 + 1..last as u32 + 1;
+                #[allow(clippy::range_plus_one)]
+                {
+                    self.range = first as u32 + 1..last as u32 + 1;
```

Comment:
I think you can change this into an inclusive range if you change the definition in `ExpandSet` to `RangeInclusive` and then change the default range in `ExpandSet::new()` to `1..=0`.

Reply:
This was something that I banged against for a while ... I don't understand why an inclusive range isn't included as an initializer for a basic `Range`. Hence, I threw up my hands and allowed it.
I'll test the change that you suggested this morning.

Reply:
Looks good ... I'll call it fixed.